prefab/curl-ssl/app/src/main/cpp/http.cpp

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (C) 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "http.h"
+
+#include <memory>
+#include <optional>
+#include <string>
+
+#include "curl/curl.h"
+
+using namespace std::string_literals;
+
+namespace {
+
+static std::string buffer;
```

Comment:
i'd be tempted to either get rid of the class and just have a C function that does everything, or at least fix this if you do intend this to be a useful bit of copy & paste?